.travis.yml

```diff
@@ -1,65 +1,34 @@
-dist: xenial
-
 env:
   global:
     # include $HOME/.local/bin for `aws`
     - PATH=$HOME/.local/bin:$PATH
 
-language: rust
-rust:
-  - stable
-  - nightly
-
-matrix:
-  allow_failures:
-    - rust: nightly
-  fast_finish: true
-
-addons:
-  apt:
-    update: true
-    snaps:
-      - node
+language: node_js
+node_js: "11"
+cache:
+  - yarn
 
 stages:
-  - name: precache
-    if: type != push OR branch = master OR tag IS present
   - name: test
-    if: type != push OR branch = master OR tag IS present
+    if: true = false
```

Comment:
If we want to skip this, putting only false doesn’t work?

Reply:
nope, wont work

Reply:
I'm keeping this as a placeholder for when I have tests, or maybe I dont need tests at all, it depends on how much/complicated js I need to write/end up writing.